### ISSUE TITLE
Can't load reveal.json

### ISSUE THREAD

**Issue by DCsunset** [webpro/reveal-md#198]
I'm using reveal-md 2.0.5.
My markdown file `slides.md` is in ~/temp/pre.
I put the reveal.json in the directory in both ~/temp and ~/temp/pre.
But the options in reveal.json still aren't loaded when I run
```
reveal-md slides.md
```

I have checked the options passed to Reveal.initialize(),
and it seems that they are still the default options.

**Comment by webpro** [comment]
Hi @DCsunset.

Are you sure `reveal.json` contains valid JSON? If so, can you try with the next command and see if there's anything interesting in the output:

```
DEBUG=reveal-md reveal-md slides.md
```

**Comment by DCsunset** [comment]
My reveal.json:
```
{
  "slideNumber": "c/t"
}

```

``` 
  reveal-md localOptions {} +2ms
  reveal-md revealOptions { slideNumber: 'c/t' } +0ms
  reveal-md { initialPath: 'slides.md',
  reveal-md   initialDir: '.',
  reveal-md   disableAutoOpen: false,
  reveal-md   highlightTheme: 'zenburn',
  reveal-md   host: 'localhost',
  reveal-md   scripts: [],
  reveal-md   css: [],
  reveal-md   preprocessor: null,
  reveal-md   port: 1948,
  reveal-md   print: false,
  reveal-md   revealOptions: {},
  ...
```

It seems that it can read the options,
but it fails to apply it.
Consequently, no slide number appears in slides.

**Comment by Allaman** [comment]
Tried to enable slide options with yaml front matter. No slide numbers in the exported file. Maybe because in in index.html the var is appearing two times

```javascript
var options = {
...
  "slideNumber": false,
...
  "slidenumber": true,
...
};
```

**Comment by DCsunset** [comment]
@Allaman Thanks. That's a good solution. Still it seems that there's some problem with reveal.json.

**Comment by dorianbrown** [comment]
Same here. I want to apply `"transition": "slide"`, and it's being read but not applied:

```
reveal-md revealOptions { transition: 'slide' } +3ms
  reveal-md { initialPath: 'test.md',
  reveal-md   initialDir: '.',
  reveal-md   highlightTheme: 'atom-one-dark',
  reveal-md   theme: 'ing.css',
  reveal-md   themeUrl: '_assets/ing.css',
  reveal-md   highlightThemeUrl: '/css/highlight/atom-one-dark.css',
  reveal-md   disableAutoOpen: false,
  reveal-md   host: 'localhost',
  reveal-md   scripts: [],
  reveal-md   css: [],
  reveal-md   preprocessor: null,
  reveal-md   port: 1948,
  reveal-md   print: false,
  reveal-md   revealOptions: {},
```

Any ideas on what to look at?

**Comment by webpro** [comment]
Released a fix in [v2.0.6](https://github.com/webpro/reveal-md/releases/tag/2.0.6).
